content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,108 @@
+---
+title: April '20 Community Gems
+date: 2020-04-16
+description: |
+  Great discussions and technical Q&A's from our users.
+descriptionLong: |
+  Look here every month for great discussions and technical Q&A's from our users 
```

Comment:
again, I think we should be trying to put a bit more information here (while preserving the main message that this is discussions from users, ... to keep it recognizable). It all affects SEO, affects link sharing snippet when you share it, etc

Reply:
good point, i'll do this in all future posts